platform/security/idp/security-idp-client/src/main/java/org/codice/ddf/security/idp/client/AssertionConsumerService.java

```diff
@@ -394,9 +394,9 @@ public Response retrieveMetadata() throws WSSecurityException, CertificateEncodi
     X509Certificate encryptionCert =
         findCertificate(systemCrypto.getEncryptionAlias(), systemCrypto.getEncryptionCrypto());
 
-    String hostname = SystemBaseUrl.getHost();
-    String port = SystemBaseUrl.getPort();
-    String rootContext = SystemBaseUrl.getRootContext();
+    String hostname = SystemBaseUrl.EXTERNAL.getHost();
```

Comment:
❗️ I think we should leave the IdP alone for now. We don't have enough independently configurable URLs for it to work behind a reverse proxy today.